Bottom: f2cb185df8813e93fb40294769b300b8dba6a144
Top:    228a1359a16a2020db0fb9e319af7ed2df238eac
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-15 20:42:57 +0000

Refresh of attempt-to-fix-broken-polygon

---

diff --git a/src/polygon1.c b/src/polygon1.c
index e720c98..8fa8686 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1703,8 +1703,8 @@ Collect (char poly, jmp_buf * e, PLINE * a, POLYAREA ** contours, PLINE ** holes
   VNODE *cur;
   DIRECTION dir = UNINITIALISED;
 
-  cur = (&a->head)        ;
-//  cur = (&a->head); //    ->next->next->next->next->next; /* Breaks circ_seg_test9.pcb */
+  cur = (&a->head);
+//  cur = (&a->head)->next->next->next->next->next; /* Breaks circ_seg_test9.pcb */
   do
     {
       // The following may be a nice speedup, but not sure if it is correct.
